.travis.yml

```diff
@@ -24,6 +24,8 @@ env:
     - TEST_DEPENDS="-rdev-requirements.txt"
     - PLAT=x86_64
     - UNICODE_WIDTH=32
+    - TWINE_USERNAME=adobe-type-tools-ci
+    # TWINE_PASSWORD is set in Travis settings
```

Comment:
@miguelsousa are you sure that setting these environment variable works with Travis deploy too? 
It does also use twine under the hood, so it should work. But I've never tried. Anyway we'll see

Reply:
actually, it failed to deploy without 'user' defined:
https://travis-ci.org/adobe-type-tools/psautohint/jobs/401042408#L878

I fixed it here: https://github.com/adobe-type-tools/psautohint/commit/65f9234828f75e57fb69d41091103a77ccf5daa8

:crossed_fingers:

Reply:
.. And also the 'password' field is required:
https://travis-ci.org/adobe-type-tools/psautohint/jobs/401053332#L878

I'm using mine just to test. The reason that alternative method works is because it calls twine directly (like we do with Appveyor), whereas in Travis we are using the `dpl` tool (which requires those settings in the .travis.yml, and in turns calls twine with them).

Reply:
👌
the afdko repo uses a script to deploy to PyPI from Travis
https://github.com/adobe-type-tools/afdko/blob/develop/.travis.yml#L117

Reply:
@miguelsousa can you please cancel this appveyor build for me?
https://ci.appveyor.com/project/adobe-type-tools/psautohint/build/1.0.256

I don't seem to have access to it, even despite I saw your invite in my email. The address is correct, not sure why it isn't working :-/
I'm logged in Appveyor with my github username @anthrotype

anyway we need to stop that appveyor build before travis pushes a new release which is going to be newer than the one appveyor is trying to push :(

Reply:
@anthrotype cancelled. Try logging into AppVeyor using your email address. The invitation is still showing up as not having been accepted.

Reply:
I did! The link asks me to "sign up" but of course I am already signed up... so I click the button to login with github and select "anthrotype" among several organizational accounts (e.g. fonttools, trufont, etc. but not afdko...), but then nothing happens. I don't seem to be able to cancel or restart the builds. Anyway, no problem. no big deal

Reply:
@anthrotype I've reconfigured the access to AppVeyor to use GH teams. Accept the invitation to https://github.com/orgs/adobe-type-tools/teams/psautohint-team and then check AppVeyor to see if you're able to restart builds now

Reply:
Thanks Miguel! Yeah, that should do it.

btw, I'm happy to report that auto-upload to pypi from travis/appveyor is now working:
https://pypi.org/project/psautohint/1.1.1a6/#files